cmd/agent.go

```diff
@@ -78,5 +78,12 @@ func init() {
 		"",
 		"Input file path, if used, it will read data from a local file instead of gathering data from clusters",
 	)
+	agentCmd.PersistentFlags().UintVarP(
+		&agent.BackoffMaxTime,
+		"backoff-max-time",
+		"",
+		60,
+		"Max time for retrying failed data gatherers. ",
```

Comment:
This should indicate the unit of the time, e.g.
```suggestion
		"Max time for retrying failed data gatherers (in seconds).",
```